packages/api/src/command/medical/customer-sequence/create-id.ts

```
@@ -1,8 +1,16 @@
+import { getEnvVar } from "@metriport/shared";
 import { OrganizationModel } from "../../../models/medical/organization";
 import { makeOrganizationOID } from "../../../shared/oid";
 
 export async function createOrganizationId(): Promise<{ oid: string; organizationNumber: number }> {
-  const maxOrgNumber = Number(await OrganizationModel.max("organizationNumber"));
+  let maxOrgNumber = Number(await OrganizationModel.max("organizationNumber"));
+
+  // If the environment variable is set, add it to the max org number
+  const organizationNumberOffset = parseInt(getEnvVar("ORGANIZATION_NUMBER_OFFSET") ?? "0");
+  if (isFinite(organizationNumberOffset)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)